zebra/rt_netlink.c

```diff
@@ -2021,6 +2021,21 @@ static int netlink_neigh_update(int cmd, int ifindex, void *addr, char *lla,
 	if (lla)
 		nl_attr_put(&req.n, sizeof(req), NDA_LLADDR, lla, llalen);
 
+	if (IS_ZEBRA_DEBUG_KERNEL) {
+		char ip_str[256];
+		struct interface *ifp =
+			if_lookup_by_index(ifindex, VRF_DEFAULT);
```

Comment:
Shouldn't we check for all VRFs? if_lookup_by_index_all_vrf()

Reply:
Thanks for your comments.
Addressed review comments in 0f9a3d765215baf8f0513a0ae2ad8b6157c31b7c
Since if_lookup_by_index_all_vrf() is a static function tried to used if_lookup_by_index_per_ns() which also gives vrf details required in ifp.

Output:
netlink_neigh_update: RTM_DELNEIGH ifname vet
h0 ifindex 5 addr ipv4 169.254.0.1 mac 82:96:20:33:25:1f vrf default(0)